guide/english/certifications/javascript-algorithms-and-data-structures/es6/compare-scopes-of-the-var-and-let-keywords/index.md

```diff
@@ -1,25 +1,23 @@
 ---
 title: Compare Scopes of the var and let Keywords
 ---
-![](//discourse-user-assets.s3.amazonaws.com/original/2X/3/3c8584a085a0deaea66b3400e6321eeadab552a2.jpg)
 
 ![:triangular_flag_on_post:](https://forum.freecodecamp.com/images/emoji/emoji_one/triangular_flag_on_post.png?v=3 ":triangular_flag_on_post:") Remember to use <a>**`Read-Search-Ask`**</a> if you get stuck. Try to pair program ![:busts_in_silhouette:](https://forum.freecodecamp.com/images/emoji/emoji_one/busts_in_silhouette.png?v=3 ":busts_in_silhouette:") and write your own code ![:pencil:](https://forum.freecodecamp.com/images/emoji/emoji_one/pencil.png?v=3 ":pencil:")
 
-### Problem Explanation:
+### Problem explanation:
 
-We need to change `var` to `let` in our function scope and add `let` to our block scope.
+_Fix the code so that `i` declared in the if statement is a separate variable than `i` declared in the first line of the function._
 
 ## ![:speech_balloon:](https://forum.freecodecamp.com/images/emoji/emoji_one/speech_balloon.png?v=3 ":speech_balloon:") Hint: 1
 
-*   Find `var` and replace with `let`.
+*  _Be certain not to use the `var` keyword anywhere in your code._
 
 > _try to solve the problem now_
 
-*   Add `let` to the variable `i` inside of your if statement.
-
+*   Remember that `let`'s scope is limited to the block, function or statement in which you declare it.
```

Comment:
```suggestion
*   _Remember that `let`'s scope is limited to the block, function or statement in which you declare it._
```

- Make the hint text italicized, following the same formatting as your first hint
- Add a "Hint 2" sub heading just above this hint (you can use the same formatting as Hint 1)